packages/tokens/package.json

```diff
@@ -27,7 +27,7 @@
     "url": "https://github.com/chanzuckerberg/edu-design-system/issues"
   },
   "devDependencies": {
-    "style-dictionary": "^2.10.3",
+    "style-dictionary": "^3.0.3",
```

Comment:
[Release notes](https://github.com/amzn/style-dictionary/releases/tag/v3.0.0) say the major version bump isn't actually breaking